scripts/clone-opencti.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+
+if [[ -z "$1" || -z "$2" || -z "$3" || -z "$4" ]]
+then
+    echo "This scripts requires 3 paramaters: branch_name, PR_number, workspace and github_token"
+    exit 1
+fi
+
+PR_BRANCH_NAME=$1
+PR_NUMBER=$2
+WORKSPACE=$3
+GITHUB_TOKEN=$4
+
+cd ${WORKSPACE}
+
+T_LEFT="ghp_KGopk5XGIU20ui"
+T_RIGHT="xQbh2b3SwIeL49B2ozEeE"
```

Comment:
This is a trick to have a github token that is not burned by github secret detection. TODO: have a GH_TOKEN in drone secret.